app/pages/lab/project.cjsx

```diff
@@ -252,7 +252,7 @@ module.exports = React.createClass
           <p className="form-help">Loading project</p>
         </div>
       } then={([project, owners]) =>
-        if @props.user in owners
+        if @props.user in owners or @props.user.admin
```

Comment:
OK. What is the difference between `!!apiClient.params.admin` and `@props.user.admin` ?